src/wazuh_mcp_server/utils/pydantic_compat.py

```
@@ -0,0 +1,132 @@
+"""
+Comprehensive Pydantic V1/V2 compatibility for v1.0.0.
+Specifically designed for Fedora compatibility while maintaining macOS/Ubuntu functionality.
+"""
+
+import sys
```

Ruff (0.11.9)

`sys` imported but unused

Remove unused import: `sys`

(F401)